exercises/luhn/src/Luhn.hs

```diff
@@ -1,11 +1,16 @@
 module Luhn (addends, checkDigit, checksum, create, isValid) where
 
-addends = error "You need to implement this function."
+addends :: Int -> [Int]
+addends n = error "You need to implement this function."
```

Comment:
>Also, I don't think it is a good idea to optimize so early.

not sure what you mean. Should I use `Integer` then, or get rid of the type declaration altogether?